src/main/java/com/novoda/notils/string/QueryUtils.java

```diff
@@ -0,0 +1,23 @@
+package com.novoda.notils.string;
+
+import java.util.Arrays;
+
+public class QueryUtils {
+
+    /**
+     * Creates a string to be used as a placeholder in {@link android.content.ContentResolver} operations selection.
+     * This will allow to use more selection arguments and have them replaced.
+     *
+     * @param size The number of selection arguments
+     * @return Example: for size == 3 : "?, ?, ?"
+     */
+    public static String createQuerySelectionPlaceholdersOfSize(int size) {
```

Comment:
the created string is for us with `IN` operators. Should be mentioned in the comment. It could also contain the operator and the brackets.

For other queries this is not a very common pattern for place holders.